tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "target": "ES5",
```

```suggestion
        "target": "es2020",
```
코딩을 하며 최근에 나온 문법들을 자주 사용할 것을 대비하여 2020버전을 추천합니다.